src/melee.cpp

```diff
@@ -978,10 +978,10 @@ bool player::valid_aoe_technique( Creature &t, const ma_technique &technique,
 
         npc * const npc_l = g->critter_at<npc>( left );
         npc * const npc_r = g->critter_at<npc>( right );
-        if( npc_l && npc_l->attitude == NPCATT_KILL ) {
+        if( npc_l && npc_l->is_enemy() ) {
```

Comment:
This change isn't a pure refactoring since `is_enemy()` returns `attitude == NPCATT_KILL || attitude == NPCATT_FLEE`. Is that intentional?